rpcs3/Emu/Cell/SPURecompiler.cpp

```diff
@@ -1636,18 +1737,24 @@ void spu_recompiler_base::dispatch(spu_thread& spu, void*, u8* rip)
 
 		atomic_storage<u64>::release(*reinterpret_cast<u64*>(rip - 8), result);
 #elif defined(ARCH_ARM64)
-		auto jump_instrs = build_function_asm<spu_function_t>("", [](native_asm& c, auto& args)
```

Comment:
Oh I see, here is the leak. I think this is the only place where it should be changed. (bytecode is much less readable and is more difficult to work with)

Reply:
I see. I've removed the other changes.

Reply:
And keep the rebuild_ubertrampoline changes as well.